amr-wind/core/FieldFillPatchOps.H

```diff
@@ -264,6 +292,92 @@ public:
     }
 #endif
 
+    void fillpatch_sibling_fields(
+        int lev,
+        amrex::Real time,
+        amrex::Array<amrex::MultiFab*, AMREX_SPACEDIM>& mfabs,
+        amrex::Array<amrex::MultiFab*, AMREX_SPACEDIM>& ffabs,
+        amrex::Array<amrex::MultiFab*, AMREX_SPACEDIM>& cfabs,
+        const amrex::IntVect& nghost,
+        const FieldState fstate = FieldState::New,
+        const FieldInterpolator itype = FieldInterpolator::FaceDivFree) override
+    {
+
+        const int ng = nghost[0];
+        const auto& geom = m_mesh.Geom(lev);
+        const auto& domain = geom.growPeriodicDomain(ng);
+        auto& sfab = m_field.state(fstate)(lev);
+        for (amrex::OrientationIter oit; oit != nullptr; ++oit) {
+            auto ori = oit();
+            const int idir = ori.coordDir();
+            const auto& dbx =
+                ori.isLow() ? amrex::adjCellLo(domain, idir, nghost[idir])
+                            : amrex::adjCellHi(domain, idir, nghost[idir]);
+
```

Comment:
to fix the abl_bndry_plane_input_native I checked for inflow BC's here and only copied them in when there is an inflow. 
`        if (bctype[ori] != BC::mass_inflow) {
            continue;
        }`
        
what do you think? or are there other BC's we should be copying face data back in?

Reply:
I thought long and hard about that one and decided that it was safest to copy everything. Clearly I was wrong. Great find! I think inflow is probably the only one that matters... I am also moving the fillpatch_siblings call in preadvect as we discussed.